apps/app/src/views/routes/Settings/components/users/UsersList.tsx

```diff
@@ -35,14 +35,32 @@ const usersQuery = graphql(/* GraphQL */ `
   query UsersListQuery {
     users {
       id
-      ...UserItemFragment
+      ...UserFragment
+      id
+      npi
+      phone
+      fax
+      email
+      address {
+        street1
+        street2
+        state
+        postalCode
+        country
+        city
+      }
       name {
+        first
         full
+        last
+        middle
+        title
       }
       roles {
+        description
+        id
```

Comment:
I dont think you need any of these fields.

Reply:
updated, extra fields removed